### PR TITLE
feat: add typography options to docs config

### PR DESCRIPTION
**Pull request by codebender828**

### CODE REVIEW

File: fern/api/definition/docs/v1/write/__package__.yml [hunk 1]
```diff
@@ -84,6 +84,19 @@ types:
       favicon: optional<FileId>
       colors: optional<ColorsConfig>
       navbarLinks: optional<list<NavbarLink>>
+      typography: optional<list<FontFamily>>
```
Comment (dsinghvi) [inline fern/api/definition/docs/v1/write/__package__.yml:87]: I think we want this to be a little more structured 

```yaml
  typography: optional<DocsTypographyConfig>

DocsTypographyConfig: 
  properties: 
    headingsFont: FontConfig
    bodyFont: FontConfig
    codeFont: FontConfig
    
 FontConfig: 
   properties: 
     name: string
     fontFile: FileId
```

Comment (dsinghvi) [reply fern/api/definition/docs/v1/write/__package__.yml:87]: i might be mistaken, but i don't think you want to ever get into an invalid state where there are 2 heading font configs? is that right?

Comment (jbakebwa) [reply fern/api/definition/docs/v1/write/__package__.yml:87]: This is also doable. I like this better actually. So each asset we upload is then given a unique key right?

Comment (jbakebwa) [reply fern/api/definition/docs/v1/write/__package__.yml]: updated

Comment (dsinghvi) [reply fern/api/definition/docs/v1/write/__package__.yml]: yup exactly

**Comment by dsinghvi** [comment]
@jbakebwa sick, the API LGTM

**Comment by dsinghvi** [comment]
To implement the API ete, you'll have to: 

- update the db definition to contain the font config as well ([link](https://github.com/fern-api/fern-definition-registry/blob/9624d9d39a63bb10d30ec18da6f21f31af0cb7e5/fern/api/definition/docs/v1/db/__package__.yml#L31C3-L39 ))
- we have a [transformToDb](https://github.com/fern-api/fern-definition-registry/blob/9624d9d39a63bb10d30ec18da6f21f31af0cb7e5/src/controllers/docs/transformDocsDefinitionToDb.ts#L9-L43) function which converts the write api -> db

**Review by dsinghvi — APPROVED** [review]
LGTM!

**Pull request merged**